src/shared/cgroup-setup.c

```diff
@@ -154,6 +154,21 @@ bool cg_is_hybrid_wanted(void) {
         return (wanted = r > 0 ? !b : is_default);
 }
 
+bool cg_legacy_force_enabled(void) {
```

Comment:
maybe add `is` to this, given that this checks the state? the other cg_is_xyz() calls are named like that after all